dash/frontend/package.json

```diff
@@ -8,7 +8,7 @@
     "start:local": "ng serve --configuration local",
     "start:dev": "ng serve --open --port 4200 --configuration dev",
     "build": "ng build",
-    "build:prod": "ng build --prod",
+    "build:prod": "ng build --configuration production",
```

Comment:
oh yeah, new version of node may have changed that

Reply:
well, of angular

Reply:
Yes, --prod was deprecated in Angular 12 & removed in Angular 14.